splice_machine/src/test/java/com/splicemachine/jdbc/JdbcApiIT.java

```diff
@@ -127,4 +112,31 @@ public void testTimeoutControl() throws Exception {
         }
     }
 
+    @Test
+    public void testUrlWithSchema() throws Exception {
+        String url = "jdbc:splice://localhost:1527/splicedb;create=true;user=%s;password=%s;schema="
+                + CLASS_NAME;
```

Comment:
I think the attribute "create=true" should not be put in the test case, it is used the very first time to create the database.